openmc/source.py

```diff
@@ -198,6 +198,43 @@ def to_xml_element(self):
             element.append(self.time.to_xml_element('time'))
         return element
 
+def plot_energy(self, n_samples=10000, seed=None, axes=None,  **kwargs):
```

Comment:
This is under-indented and that's why the tests are failing
```suggestion
    def plot_energy(self, n_samples=10000, seed=None, axes=None,  **kwargs):
```

Reply:
Thanks Remi